src/odemis/driver/xt_client.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
```

Comment:
No shebang needed in a module, as it's no possible to execute directly => drop the line.